yeonjae22/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/view/product/ProductActivity.kt

```diff
@@ -18,15 +18,15 @@ class ProductActivity : AppCompatActivity() {
         setContentView(R.layout.activity_product)
 
         val productRepository = ProductRepository(
-            ProductRemoteDataSource(),
-            ProductLocalDataSource()
+            ProductRemoteDataSource.getInstance(),
```

Comment:
```suggestion
            ProductRemoteDataSource.getInstance(
                      RetrofitInstance.getInstance<ElevenStreetApi>("https://openapi.11st.co.kr/openapi/")
                      ),
```
위의 리뷰를 수정하고 이렇게 넣어주면 되겠죠?